java-network-security/google-cloud-network-security/src/main/java/com/google/cloud/networksecurity/v1beta1/NetworkSecurityClient.java

```diff
@@ -471,7 +508,7 @@ public final AuthorizationPolicy getAuthorizationPolicy(GetAuthorizationPolicyRe
    */
   public final OperationFuture<AuthorizationPolicy, OperationMetadata>
       createAuthorizationPolicyAsync(
-          AuthorizationPolicyName parent,
+          LocationName parent,
```

Comment:
This v1beta1 API changed the type in the argument. This is also there in the split repo.

<img width="1165" alt="Screen Shot 2022-07-25 at 12 24 06 PM" src="https://user-images.githubusercontent.com/28604/180827560-a57021c5-952d-481f-ba08-960476f87fc9.png">